packages/google-cloud-access-context-manager/docs/type.rst

```diff
@@ -0,0 +1,6 @@
+google.identity.accesscontextmanager.type
+=========================================
+
+.. automodule:: google.identity.accesscontextmanager.type
```

Comment:
A bunch of these `type.rst` files used to have these lines ending in `….type.XXXX_pb2`, but now they're truncated to end in `….type`. I take it this is expected?

Reply:
Yes. This is intended. For now the docs for `pb2.py` files are not meaningful at the moment. We may use `pb2.pyi` files instead for docs.  b/375167345 is the tracking bug to improve docs in `pb2.pyi`. The protobuf team has indicated that docs (comments) won't be added to `pb2.py` files.